ktor-server-rate-limiting/README.md

```diff
@@ -24,48 +31,69 @@ To apply the `RateLimitingPlugin`, you need to `install` it in your Ktor route a
 
 ```kotlin
 routing {
-    route("limited-route") {
-        limit = 100
-        timeWindow = 1.minutes
-        burstLimit = 10
-        whiteListedHosts = setOf("trusted-host.com")
-        blackListedAgents = setOf("malicious-agent")
-        rateLimitExceededCallHandler = { call, count ->
-            call.respond(HttpStatusCode.TooManyRequests, "Rate limit exceeded: call count: $count, limit: $limit")
-        }
-        logRateLimitHits = true
+  route("limited-route") {
```

Comment:
Can you globally define it too?

Reply:
this is just an example. the route is basically the URL path, in this case it'd be https://host/limited-route

Reply:
Got it. Question is whether there is an example for global use - for every path. 

In this case you might consider allowing per route opt out.

Reply:
The earlier version had a global option, which i removed in this version the rate limiting relies on authentication, which is a route scoped plugin, and so it is not possible to respond to the authentication hook in the global pipeline. 

Setting this (route scoped) on any route will apply it to any nested routes, so:

```
route("a"){
   install(rateLimiting){}
      route("b"){}
}
```
will apply it to `.../a` and `.../a/b`